client/src/ui/MultiPlayerSelectionWrapper.lua

```diff
@@ -19,7 +19,9 @@ StackPanel)
 function MultiPlayerSelectionWrapper:addElement(uiElement, player)
   self.wrappedElements[player] = uiElement
   uiElement.yieldFocus = function()
-    self.yieldFocus()
+    if self.yieldFocus then
+      self.yieldFocus()
+    end
```

Comment:
I don't see a scenario in which this is not an optionally focused element of a bigger selection but maybe you had a certain idea here?
I feel like with this change, if I forgot to assign the function for your specific wrapper, it's just going to silently not do what I am expecting it to with no immediate indication why. I think with the way this element is built, it's better if this explicitly crashes if I did not assign a yield behaviour.